youtube_dl/extractor/youmaker.py

```diff
@@ -0,0 +1,144 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from .. import utils
```

Comment:
Every existing extractor imports individual functions rather than the whole file.

Reply:
ok